governance/xc_admin/packages/xc_admin_frontend/utils/parseEvmExecuteCallData.ts

```diff
@@ -0,0 +1,68 @@
+import { decodeFunctionData } from 'viem'
+
+const ABI = [
+  {
+    inputs: [],
+    name: 'acceptOwnership',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [],
+    name: 'acceptAdmin',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'newImplementation',
+        type: 'address',
+      },
+    ],
+    name: 'upgradeTo',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+]
```

Comment:
you should add `as const` here to benefit from all of the viem typescript support. The code in the parse function will become much simpler since the type hints become stronger